src/pages/core/plugins/edition.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Edition Plugin
+metaTitle: Core - Edition Plugin
+description: Learn about the MPL Core Edition Plugin
+---
+
+The Attribute Plugin is a `Authority Managed` plugin that stores an Edition Number within the asset. Together with the soon to be added Master Edition Plugin those Editions could be compared to the [Edition concept in Metaplex Token Metadata](/token-metadata/print).
+
+The Attribute Plugin will work in areas such as:
+
+- Prints of the same Asset
+
+We recommend to use Candy Machine to create and sell your editions. It will handle numbering automatically. 
+
+## Works With
+
+|                     |     |
+| ------------------- | --- |
+| MPL Core Asset      | ✅  |
+| MPL Core Collection | ❌  |
+
+## Arguments
+
+| Arg           | Value                               |
+| ------------- | ----------------------------------- |
+| number        | number                              |
+
+The number is a specific value that is assigned to the asset. Usually this number is unique.
+
+## Creating an Asset with the editions plugin
+
+The Editions Plugin must be added on creation of the asset. As long as it is mutable the number can be changed.
+
+{% callout type="note" %}
+
+The Creator should make sure that a number is not used twice. If you are using Candy Machine with the edition Guard this is handled automatically.
```

Comment:
can we also mention that the intended usage is with core candy machine with the edition guard as well as master edition plugin on the colleciton?

Reply:
For sure - please check again.